src/kibana/components/index_patterns/_field.js

```diff
@@ -38,7 +38,7 @@ define(function (require) {
 
       var indexed = !!spec.indexed;
       var scripted = !!spec.scripted;
-      var sortable = indexed && type.sortable;
+      var sortable = spec.name === '_score' || (indexed && type.sortable);
```

Comment:
I agree with you in principle, but there really is no other field like `_score` (in that they are always sortable) and I feel like whatever property we add to the spec will equal `spec.name === '_score'` anyway.